pkg/baseline/packageIndex.json

```diff
@@ -43,8 +57,12 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
         "4.0.1.0": "4.0.1",
-        "4.0.2.0": "4.3.1",
```

Comment:
I would prefer keeping the mapping of 4.3.1 (the latest release) and leaving 4.3.0 in comments.

Reply:
Checking with @ericstj to make sure there wasn't a specific reason for using the first pkg where the assembly first appeared. It may have some impact on package validation.

Reply:
According to Eric...
`The purpose of the version table in the package index is to tell the packaging infrastructure the minimum package version needed for a particular assembly version.`